worker/worker.go

```diff
@@ -1373,7 +1373,7 @@ func (cl *contractLock) keepaliveLoop() {
 	}
 }
 
-func (w *worker) AcquireContract(ctx context.Context, fcid types.FileContractID, priority int) (_ contractReleaser, err error) {
+func (w *worker) acquireContract(ctx context.Context, fcid types.FileContractID, priority int) (_ revisionUnlocker, err error) {
```

Comment:
Should we consider a full rename to revision locks rather than contract locks?

Reply:
I did actually consider a refactor like that for a f/u. Basically to unify locking an fetching a revision in all the places. So that you only need to call `withRevision` and get an implicitly locked revision. I can rename this to `acquireRevision` in the process.